File: PSET1/mario_more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("h: ");
    }
    while (!(h > 0 && h < 9));

    for (int i = 1; i <= h; i++)
    {
        // left half Mario's Pyramid
        for (int j = 0; j < h; j++)
        {
            printf((j < h - i) ? " " : "#");
        }
        printf("  ");
        // right half Mario's Pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: PSET6/Mario_more/mario.py
from cs50 import get_int
h = get_int("Height: ")
while not 0 < h < 9:
    h = get_int("Height: ")

print(*list(map(lambda i: (h-i)*" "+i*"#"+"  "+i*"#", range(1, h+1))), sep="\n")



